Source/DafnyCore/AST/Grammar/ParseErrors.cs

```diff
@@ -126,7 +126,7 @@ public enum ErrorId {
 that is a single underscore is used as a wild-card match.
 ", range => new List<DafnyAction> { new("remove underscore",
     new [] {
-      new DafnyCodeActionEdit(range, range.PrintOriginal().Substring(1))
+      new DafnyCodeActionEdit(range.ToDafnyRange(), range.PrintOriginal().Substring(1))
```

Comment:
Could you perhaps add an overload of the constructor so that you don't need to call ToDafnyRange on the first argument? Seems like you do this a lot.